src/dpp/sslclient.cpp

```diff
@@ -373,6 +373,10 @@ void ssl_client::on_read(socket fd, const struct socket_events& ev) {
 }
 
 void ssl_client::on_write(socket fd, const struct socket_events& e) {
+	/* We wanted write before so keep it */
+	socket_events se{e};
+	se.flags |= WANT_WRITE;
+	owner->socketengine->update_socket(se);
```

Comment:
really not sure about having this here. is it needed to fix the voice stuff? this will make the socket engine spin on 100% CPU always